docs/HC-12--Transmit.md

```diff
@@ -0,0 +1,4 @@
+# XBee HC-12
+
+https://www.allaboutcircuits.com/projects/understanding-and-implementing-the-hc-12-wireless-transceiver-module/
```

Comment:
HC-12 is not an Xbee module to my understanding. Xbee is a specific brand & their proprietary protocol(s). We should not call it that way

Comment has not been resolved before. To